manifests/4.10/kubernetes-nmstate-operator.v4.10.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,233 @@
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: kubernetes-nmstate-operator
+                name: kubernetes-nmstate-operator
+            spec:
+              containers:
+              - command:
+                - manager
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: OPERATOR_NAME
+                  value: kubernetes-nmstate-operator
+                - name: ENABLE_PROFILER
+                  value: "False"
+                - name: PROFILER_PORT
+                  value: "6060"
+                - name: RUN_OPERATOR
+                - name: HANDLER_IMAGE
+                  value: quay.io/openshift/origin-kubernetes-nmstate-handler:4.10
+                - name: HANDLER_IMAGE_PULL_POLICY
+                  value: Always
+                - name: HANDLER_NAMESPACE
+                  value: openshift-nmstate
+                image: quay.io/openshift/origin-kubernetes-nmstate-operator:4.10
+                imagePullPolicy: Always
+                name: nmstate-operator
+                resources: {}
+              nodeSelector:
+                node-role.kubernetes.io/master: ""
+              serviceAccountName: nmstate-operator
+              tolerations:
+              - effect: NoSchedule
+                key: node-role.kubernetes.io/master
+                operator: Exists
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          - pods
+          verbs:
+          - '*'
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - '*'
+        - apiGroups:
+          - policy
+          resources:
+          - poddisruptionbudgets
+          verbs:
+          - '*'
+        serviceAccountName: nmstate-operator
+      - rules:
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - use
+          resourceNames:
+          - privileged
+        serviceAccountName: nmstate-handler
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - nmstate
+  - networking
+  - NetworkManager
+  links:
+  - name: Kubernetes Nmstate Operator
+    url: https://github.com/nmstate/kubernetes-nmstate
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat
+  maturity: beta
```

Comment:
At some point this will need to be updated since we've moved the API to v1 and are planning to call this GA in 4.10.

Reply:
Good point. I didn't really find valid values for this field as it is a simple string (https://github.com/operator-framework/operator-sdk/blob/67ac8204f15ba3164308f41404b184c5f120d3e0/internal/generate/clusterserviceversion/bases/clusterserviceversion.go#L53). I now set it to `GA`.